packages/dapp/src/components/MultisigWalletHeaderAlert/MultisigWalletHeaderAlert.tsx

```diff
@@ -10,17 +10,18 @@ import {
 
 export const MultisigWalletHeaderAlert = () => {
   const { t: translate } = useTranslation();
+  const I18_PATH = 'multisig.alert.';
 
   return (
     <MultisigWalletHeaderAlertContainer>
       <MultisigWalletHeaderAlertTitle>
         <InfoRoundedIcon />
         <Typography fontWeight={700} marginLeft={1}>
-          {translate(`multisig.alert.title`)}
+          {translate(`${I18_PATH}multisig.alert.title`, 'translation')}
```

Comment:
Are we sure that the full path is `multisig.alert.multisig.alert.title`? 🙂

Reply:
Oh, oh.. That must be wrong! Fixed now